sdk/attestation/Azure.Security.Attestation/src/Models/PolicyCertificateModification.cs

```diff
@@ -20,24 +20,12 @@ internal partial class PolicyCertificateModification
         /// <param name="bodyCertificate"><see cref="X509Certificate2"/> to be encoded as a JSON Web Key in the body of the token.</param>
         internal PolicyCertificateModification(X509Certificate2 bodyCertificate)
         {
-            this.InternalPolicyCertificate = new JsonWebKey(
-                    alg: "RS256",
-                    kty: "RSA",
-                    use: "sig",
-                    x5C: new string[] { Convert.ToBase64String(bodyCertificate.Export(X509ContentType.Cert)) },
-                    crv: "",
-                    d: "",
-                    dp: "",
-                    dq: "",
-                    e: "",
-                    k: "",
-                    kid: "",
-                    n: "",
-                    p: "",
-                    q: "",
-                    qi: "",
-                    x: "",
-                    y: "");
+            this.InternalPolicyCertificate = new JsonWebKey(kty: "RSA")
```

Comment:
```suggestion
            InternalPolicyCertificate = new JsonWebKey(kty: "RSA")
```

As sad as it makes me, we've been asked to avoid prefixing with `this.`